pyproject.toml

```diff
@@ -37,7 +37,7 @@ lxml = "^4.9.1"
 plotly = "^5.11.0"
 termcolor = "^2.2.0"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
```

Comment:
This is how the official documentation writes this section. 

https://github.com/python-poetry/poetry?tab=readme-ov-file#poetry-python-packaging-and-dependency-management-made-easy